zipline/algorithm.py

```diff
@@ -56,7 +56,7 @@
     OrderInBeforeTradingStart)
 from zipline.finance.trading import TradingEnvironment
 from zipline.finance.blotter import Blotter
-from zipline.finance.commission import PerShare, PerTrade, PerDollar
+from zipline.finance.commission import PerShare, CommissionModelBase
```

Comment:
Most of our other base classes leave out `Base` in the name (e.g. `PipelineLoader`, `TradingControl`, `ExecutionStyle`).